docs/reference/reana-yaml/index.md

```diff
@@ -64,6 +64,18 @@ The **inputs** property is composed of:
 | directories | *list*       | *optional* | Lists all the input directories to the workflow. Will be seeded to the workspace before running.                                 |
 | files       | *list*       | *optional* | Lists all the input files to the workflow.  Will be seeded to the workspace before running.                                      |
 | parameters  | *dictionary* | *optional* | Specifies all the input parameters to the workflow. It is a dictionary of parameter names and their values expressed as strings. |
+| options     | *dictionary* | *optional* | Specifies operational options for each workflow engine, see below.                                                               |
+
+The **inputs.options** property describes operational options that can be used for the different workflow engines. The available options are:
+
+| Property | Type     | Mandatory?  | Workflow engine | Description                            |
+| -------- | ----     | ----------  | ----------------|                                        |
+| CACHE    | *string* | *optional*  | Serial          | Whether the workflow engine should cache the results of each step for faster execution of subsequent workflow runs. Disabled by default. Can be `on` or `off`. |
+| FROM     | *string* | *optional*  | Serial          | Allows partial execution of a workflow from the `FROM` step, including it. It can be combined with `TARGET`. The value must be the name of the step. |
```

Comment:
Allows partial execution of a workflow starting from a desired step (including it) passed as a value. The `FROM` option can be combined with the `TARGET` option.